drivers/utils.go

```diff
@@ -47,3 +49,10 @@ func GenerateAdvisoryLockID(databaseName, schemaName string) (string, error) {
 	sum = sum * uint32(advisoryLockIDSalt)
 	return fmt.Sprint(sum), nil
 }
+
+func (c *Config) GetContext() (context.Context, context.CancelFunc) {
```

Comment:
Nit: this feels better as a function rather than a method. `ctx, cancel := GetContext(timeout)` looks neater. 0/5.